src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
+@use "./utils/variables" as *;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  margin: 0 auto;
+  height: 100vh;
+  width: 7 * $cell-size + 2 * $container-paddings + $cell-shift * 6;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-shift;
+  padding: 0 10px;
+
+  &__date {
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    background-color: #eee;
+    text-align: center;
+    line-height: $cell-size;
+  }
+
+  @for $index from 1 through 31 {
+    &__date--#{$index}::before {
+      content: "#{$index}";
+    }
+  }
+
+  @for $days from 28 through 31 {
+    &--month-length--#{$days - 1}
+    &__date:nth-child(n + #{$days}) {
+      display: none;
+    }
+  }
+
+  @each $day-name, $day-count in $week-day {
+    &--start-day--#{$day-name}
+    &__date:first-child {
+      margin-left: ($cell-size + $cell-shift) * $day-count;
+    }
+  }
+
+  &__date:hover {
+    background-color: pink;
+    transform: translateY(-20px);
+    cursor: pointer;
+    transition: background-color 0.5s, transform 0.5s;
```

Comment:
Same with transition time.
Btw, the transition property should be set for the normal state of the element to work properly.
Currently, it works strangely because of that.
![hovers](https://user-images.githubusercontent.com/96409563/201892697-25acd4d8-3b47-4bef-9f66-e3fc7fe4eb1c.gif)